projects/packages/sync/src/modules/class-themes.php

```diff
@@ -192,7 +190,7 @@ public function detect_theme_edit( $redirect_url ) {
 		$query_params = array();
 		wp_parse_str( $url['query'], $query_params );
 		if (
-			! isset( $_POST['newcontent'] ) ||
+			! isset( $_POST['newcontent'] ) || // phpcs:disable WordPress.Security.NonceVerification.Missing -- We are using $_POST['newcontent'] to detect if this is a theme edit during a redirect.
```

Comment:
Same, I like being explicit about these 🙂
```suggestion
			! isset( $_POST['newcontent'] ) || // phpcs:disable WordPress.Security.NonceVerification.Missing -- 'wp_redirect' gets fired for a lot of things. We're only using $_POST['newcontent'] to limit action to redirects from theme edits, we're not doing anything with or in response to the post itself.
```